tidy3d/components/spice/components/capacitor.py

```diff
@@ -0,0 +1,18 @@
+from typing import Optional
```

Comment:
I understand how I could reuse everything in the `components` folder, but for the RF solver I don't see how I would make use of the classes in `analysis` or `sources`. At some point if we support cosimulation with spice it would make more sense, but at the moment I think the main operation would be to extract S parameters from device using Tidy3D and then using the S parameter model of the device in a larger circuit.